Scripts/Proxies/TwoHandedProxyBase.cs

```diff
@@ -74,34 +73,56 @@ public virtual bool hidden
 			}
 		}
 
-		private bool m_Hidden;
-
 		public Dictionary<Transform, Transform> menuOrigins { get; set; }
 		public Dictionary<Transform, Transform> alternateMenuOrigins { get; set; }
 		public Dictionary<Transform, Transform> previewOrigins { get; set; }
 		public Dictionary<Transform, Transform> fieldGrabOrigins { get; set; }
 
+		// Local method use only -- created here to reduce garbage collection
+		static readonly List<Tooltip> k_TooltipList = new List<Tooltip>();
+
 		public virtual void Awake()
 		{
 			m_LeftHand = ObjectUtils.Instantiate(m_LeftHandProxyPrefab, transform).transform;
 			m_RightHand = ObjectUtils.Instantiate(m_RightHandProxyPrefab, transform).transform;
 			var leftProxyHelper = m_LeftHand.GetComponent<ProxyHelper>();
 			var rightProxyHelper = m_RightHand.GetComponent<ProxyHelper>();
 
-			m_ProxyMeshRoots.Add(leftProxyHelper.meshRoot);
-			m_ProxyMeshRoots.Add(rightProxyHelper.meshRoot);
+			foreach (var tooltip in leftProxyHelper.rightTooltips)
```

Comment:
Should probably encapsulate this so the we don't have to have nearly the same code twice in a row